posthog/migrations/0061_featureflag.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.0.7 on 2020-06-17 08:33
+
+from django.conf import settings
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('posthog', '0060_auto_20200616_0746'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FeatureFlag',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=400)),
+                ('key', models.CharField(max_length=400)),
+                ('filters', django.contrib.postgres.fields.jsonb.JSONField(default=dict)),
```

Comment:
What does the filters do here?

Reply:
Filters contains the properties you can filter users on.